modules/statistics/php/stats_demographic.class.inc

```diff
@@ -25,6 +25,15 @@ namespace LORIS\statistics;
  */
 class Stats_Demographic extends \NDB_Form
 {
+
+    /**
+     * The default params
+     *
+     * @var    array
+     * @access private
+     */
+    var $params = null;
```

Comment:
This is PHP4 style. The `@access private` should be removed, and use the language level private/protected/public keywords instead of `var` in the code rather than a comment.

(This should also probably be protected, not private.)